pkg/config/config.go

```diff
@@ -77,7 +77,9 @@ func LoadConfigFromString(configString string) (*Config, error) {
 	if config.Matrix.UserID == "" ||
 		config.Matrix.HomeserverURL == "" ||
 		config.Matrix.AccessToken == "" ||
-		config.Conference.KeepAliveTimeout == 0 {
+		config.Conference.KeepAliveTimeout == 0 ||
+		config.Conference.KeepAliveTimeout > 30 ||
+		config.Conference.PingInterval < 30 {
```

Comment:
At some point I'd suggest splitting these out into multiple checks so we can print an error message that tells you what's wrong, otherwise this could get unwieldy, but doesn't have to be now.

Reply:
Have added a TODO and made https://github.com/matrix-org/waterfall/issues/71